Source/WebKit/Shared/WebPushDaemonConnectionConfiguration.h

```diff
@@ -32,7 +32,7 @@
 namespace WebKit::WebPushD {
 
 struct WebPushDaemonConnectionConfiguration {
-    std::optional<Vector<uint8_t>> hostAppAuditTokenData;
+    Vector<uint8_t> hostAppAuditTokenData;
```

Comment:
Could this be audit_token_t?

Reply:
I seem to remember that this would require more changes in encoding glue than we wanted to deal with when we originally made this object. I can look at this in a future patch.